### PR TITLE
major refactor; rename to azureml-cheatsheets

### PR DESCRIPTION
**Pull request by lostmygithubaccount**
- update license, code of conduct
- saving
- final saving

# PR into Azure/azureml-cheatsheets

## Checklist

I have:

- [x] read and followed the [contributing guidelines](https://azure.github.io/azureml-cheatsheets/docs/misc/contributing)

## Changes

*This PR assumes changing the repository name to `azureml-cheatsheets` to align with long term goals*

- replace "azureml-web" with "azureml-cheatsheets" where applicable
- move contributing guidilines from readme to website
- edits to PR templates and issue templates
- remove unused docs (cbdocs, others)
- reoganize docs into docs/cheatsheets (main content) and docs/misc (side stuff like contributing guide)
- restructure nav bar per interface e.g. Python, CLI, etc.
    - docs/cheatsheets structured the same way
- edits to sidebar
- minor edits to License
- add code of conduct
- rename `deploy-website` to `deploy`
- remove the "features" from the front page
- slightly edit tag line
- minor edits to footer

Overall, the v1 Python SDK cheat sheet content is untouched. 

fixes #
closes #45 (same fix made in here)

### CODE REVIEW

**Comment by lostmygithubaccount** [comment]
I think we'd need to:

- [ ] review this PR (@aminsaied)
- [ ] merge; rename repo to `azureml-cheatsheets` 
- [ ] make sure nothing broke

Then in a later PR we can remove docs/vs-code-snippets

File: README.md [hunk 1]
```diff
@@ -1,74 +1,27 @@
-# Azure ML Cheat Sheet
+# Azure Machine Learning cheat sheets
```
Comment (aminsaied) [inline README.md:1]: Capitalize first letters for consistency

File: website/docs/misc/deployment.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: Deployment
+description: One-time website deployment setup.
+---
+
+## Deployment
+
+This article describes the one-time process for deploying this website.
+
+This repo has GitHub actions in place that automate deployment by watching the `website` branch.
```
Comment (aminsaied) [inline website/docs/misc/deployment.md:10]: It watches the "main" branch

File: website/src/pages/index.js [hunk 3]
```diff
@@ -8,36 +8,6 @@ import styles from './styles.module.css';
 
 const features = [
   {
```
Comment (aminsaied) [inline website/src/pages/index.js:10]: It looks strange with no content here. When we have multiple cheatsheets we should link to them from here, but for now I propose we keep the quick links to the v1 pages. It just looks unfinished to have nothing here.

File: website/docusaurus.config.js [hunk 4]
```diff
@@ -2,49 +2,49 @@ const path = require('path');
...
+        alt: 'Logo',
         src: 'img/logo.svg',
       },
       items: [
-        {to: 'docs/cheatsheet/', label: 'Cheat Sheet', position: 'left'},
-        {to: 'docs/vs-code-snippets/snippets', label: 'Snippets', position: 'left'},
-        // {to: 'docs/templates/', label: 'Templates', position: 'left'},
-        // {to: 'blog', label: 'Blog', position: 'left'},
-        // {href: 'https://github.com/Azure/azureml-web', label: 'GitHub', position: 'right',
+        {to: '/docs/cheatsheets/python/v1/cheatsheet', label: 'Python', position: 'left'},
```
Comment (aminsaied) [inline website/docusaurus.config.js:20]: Not sure about "Python". What about "Python SDK"?

File: website/sidebars.js [hunk 5]
```diff
@@ -1,49 +1,34 @@
 module.exports = {
-  mainSidebar: {
-    'Menu': [
+  pythonSidebar: {
+    'Python': [
```
Comment (aminsaied) [inline website/sidebars.js:3]: Not sure about "Python". What about "Python SDK"?

**Review by aminsaied — CHANGES_REQUESTED** [review]

File: website/src/pages/index.js (hunk 3 above)
Comment (lostmygithubaccount) [reply website/src/pages/index.js:10]: added back the old content, modifying to new paths to the docs - updating this if needed can happen in a following PR

**Review by aminsaied — APPROVED** [review]

**Pull request merged**